src/services/UserService.ts

```diff
@@ -55,21 +55,25 @@ const cachedTimeZone: CachedTimeZone = {
  * get current/active user
  */
 const getUser = async (): Promise<UserResponse> => {
-  const response: UserResponse = await httpCurrentUser.get<UserServerResponse, UserData>({}, (data) => ({
-    user: data?.user
-      ? {
-          countryCode: data.user.countryCode,
-          email: data.user.email,
-          emailNotificationsEnabled: data.user.emailNotificationsEnabled,
-          firstName: data.user.firstName,
-          globalRoles: data.user.globalRoles,
-          id: data.user.id,
-          lastName: data.user.lastName,
-          locale: data.user.locale,
-          timeZone: data.user.timeZone,
-        }
-      : undefined,
-  }));
+  const response: UserResponse = await httpCurrentUser.get<UserServerResponse, UserData>({}, (data) => {
+    return {
```

Comment:
Is this necessary?

Reply:
Yeah this change is necessary to include the new cookie consent properties in the user object